src/DataSourceProvider.vue

```diff
@@ -4,11 +4,14 @@
       <strong>{{ widgetData.dataSourceTitle || 'Select a data source' }}</strong>
     </div>
 
-    <div v-if="isLoading" class="spinner-holder animated">
+    <div v-show="isLoading" class="spinner-container animated">
```

Comment:
what's the reason for using `show` instead of `if`?

Reply:
@squallstar Forgot to return from the last fix attempt